client/post-editor/editor-action-bar/index.jsx

```diff
@@ -82,4 +88,18 @@ export default React.createClass( {
 			</div>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const postId = getEditorPostId( state );
+		const post = getEditedPost( state, siteId, postId );
+
+		return {
+			siteId,
+			postId,
+			post
+		};
+	},
+)( EditorActionBar );
```

Comment:
I'm seeing another regression due to the reduxification here. The "Author" switcher is not updating the author correctly when we select a different one until we save. (That's probably because it's receiving the Redux Author while it performs the changes on Flux)